From 27f60046200793f94f140774c068692016f3d1e2 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Fri, 4 May 2018 18:20:28 +0100
Subject: [PATCH] common/cfgfiles: Apply Solus defaults

At this point we're only defaulting to having coloured nicks.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/common/cfgfiles.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/common/cfgfiles.c b/src/common/cfgfiles.c
index fdee9f2..1402999 100644
--- a/src/common/cfgfiles.c
+++ b/src/common/cfgfiles.c
@@ -797,6 +797,7 @@ load_default_config(void)
 	prefs.hex_text_stripcolor_topic = 1;
 	prefs.hex_text_thin_sep = 1;
 	prefs.hex_text_wordwrap = 1;
+	prefs.hex_text_color_nicks = 1;
 	prefs.hex_url_grabber = 1;
 
 	/* NUMBERS */
-- 
2.16.3

